teuthology/task/selinux.py

```diff
@@ -91,8 +100,13 @@ def get_denials(self):
         all_denials = dict()
         ignore_known_denials = '\'\(' + 'comm="dmidecode"'
```

Comment:
This should probably be a ``list``. You can then use ``list.join()`` in combination with ``str.format()`` to assemble the regex